app/webpacker/stylesheets/patterns/_pagination.scss

```diff
@@ -0,0 +1,100 @@
+// TODO rewrite this to use more of GOVUK Frontend  mixin and styling
```

Comment:
Or this... we have nothing to paginate yet and we have components for this stuff now.

Reply:
I worry this'll break the GIAS stuff, but ya ... that's not a good enough reason to keep it.

Reply:
Removed, beyond scope.